src/entrypoints/main/router/validator/pages/validator-page/index.tsx

```diff
@@ -18,19 +20,43 @@ const ValidatorPage: FC<Props> = ({
   isValidating,
   validatorActions: { validateRdfRequested: validateRdf }
 }) => {
-  const onValidate = (inputFile: File | string) => {
-    validateRdf({
-      resource: inputFile,
-      version: DcatVersion.V2
-    });
-  };
```

Comment:
Denne kan godt beholdes, fordi den kan brukes to steder - i useEffect og i validation input form